### DISCUSSION TITLE
Does ChistaDATA Cloud offer backup and disaster recovery services?

### DISCUSSION THREAD

**Discussion by emoido** [ChistaDATA/Cloud#10] [❓FAQ] [answered]
Does ChistaDATA Cloud offer backup and disaster recovery services?

**Comment by TheraniAA** [comment, marked as answer]
Yes, ChistaDATA Cloud offers backup and disaster recovery services for ClickHouse® databases. Automated backups are created at regular intervals according to a user-specified schedule, and the backups are stored in secure, geographically redundant locations to ensure data durability. In addition, ChistaDATA Cloud offers disaster recovery capabilities through automatic failover to backup replicas in case of a primary replica failure. This helps to minimize downtime and ensure continuity of service for ClickHouse® users.
